.pre-commit-config.yaml

```diff
@@ -56,12 +56,31 @@ repos:
     -   id: incorrect-sphinx-directives
         name: Check for incorrect Sphinx directives
         language: pygrep
-        entry: >-
-            \.\. (autosummary|contents|currentmodule|deprecated
-            |function|image|important|include|ipython|literalinclude
-            |math|module|note|raw|seealso|toctree|versionadded
-            |versionchanged|warning):[^:]
+        entry: "\
```

Comment:
probably cleaner to do it with (?x) : https://pre-commit.com/#regular-expressions